tests/rust/tcp-test/close/wait.rs

```diff
@@ -224,3 +227,119 @@ fn wait_after_async_close_connecting_socket(libos: &mut LibOS, remote: &SocketAd
 
     Ok(())
 }
+
+fn wait_on_invalid_queue_token_returns_einval(libos: &mut LibOS) -> Result<()> {
```

Comment:
Could you add a one-line comment about this test here? You may want to copy what we have for description on the issue.